setup.py

```diff
@@ -0,0 +1,100 @@
+#
+# Flask-Menu is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+import os
+import re
+import sys
+
+from setuptools import setup
+from setuptools.command.test import test as TestCommand
+
+
+class PyTest(TestCommand):
+
+    user_options = [('pytest-args=', 'a', 'Arguments to pass to py.test')]
+
+    def initialize_options(self):
+        TestCommand.initialize_options(self)
+        try:
+            from ConfigParser import ConfigParser
+        except ImportError:
+            from configparser import ConfigParser
+        config = ConfigParser()
+        config.read("pytest.ini")
+        self.pytest_args = config.get("pytest", "addopts").split(" ")
+
+    def finalize_options(self):
+        TestCommand.finalize_options(self)
+        self.test_args = []
+        self.test_suite = True
+
+    def run_tests(self):
+        # import here, cause outside the eggs aren't loaded
+        import pytest
+        errno = pytest.main(self.pytest_args)
+        sys.exit(errno)
+
+# Get the version string. Cannot be done with import!
+with open(os.path.join('flask_notifications', 'version.py'), 'rt') as f:
+    version = re.search(
+        '__version__\s*=\s*"(?P<version>.*)"\n',
+        f.read()
+    ).group('version')
+
+tests_require = [
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
+    'pytest-pep8>=1.0.6',
+    'pytest>=2.6.1',
+    'coverage<4.0a1'
+]
+
+setup(
+    name='Flask-Notifications',
+    version=version,
+    url='https://github.com/inveniosoftware/flask-notifications',
+    license='BSD',
+    author='jvican',
+    author_email='info@invenio-software.org',
+    description='Flask-Notifications is a Flask extension that adds support '
+        'for notifications.',
+    long_description=open('README.rst').read(),
+    packages=['flask_notifications'],
+    zip_safe=False,
+    include_package_data=True,
+    platforms='any',
+    install_requires=[
+        'Flask',
+        'six',
+        'celery[redis]',
+        'Flask-Mail'
+    ],
+    extras_require={
+        'docs': ['sphinx'],
+    },
+    tests_require=tests_require,
+    cmdclass={'test': PyTest},
+    classifiers=[
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Framework :: Flask',
+        'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Development Status :: 5 - Production/Stable'
```

Comment:
maybe we should change the classifier to `'Development Status :: 1 - Planning',`